lua/packer.lua

```diff
@@ -20,6 +20,7 @@ local config_defaults = {
   transitive_opt = true,
   transitive_disable = true,
   auto_reload_compiled = true,
+  rplugin_load_event = 'BufReadPre',
```

Comment:
This is different to how `dein` handles loading remote plugins since a user doesn't specify an event instead, as far as I can tell, remote plugins are set up when the first lazy loaded plugin is loaded and then a `g:_dein_rplugins_loaded` var is set and used a guard against future runs. Not sure if this is better or worse, it seems more automatic 🤷🏿‍♂️ so can change it to work like that